mobile/lib/features/trade/trade_dialog.dart

```diff
@@ -63,55 +68,89 @@ Widget createSubmitWidget(
       break;
   }
 
-  Column body = Column(
-    mainAxisSize: MainAxisSize.min,
-    children: [
-      SizedBox(
-        width: 200,
-        child: Wrap(
-          runSpacing: 10,
-          children: [
-            pendingOrder.positionAction == PositionAction.close
-                ? ValueDataRow(type: ValueType.amount, value: pendingOrder.pnl, label: pnlText)
-                : ValueDataRow(
-                    type: ValueType.amount, value: pendingOrderValues?.margin, label: "Margin"),
-            ValueDataRow(
-                type: ValueType.amount, value: pendingOrderValues?.fee ?? Amount(0), label: "Fee")
-          ],
-        ),
-      ),
+  List<Widget> children = [];
+  if (pendingOrder.failureReason != null) {
+    children.add(
       Padding(
         padding: const EdgeInsets.only(top: 20, left: 10, right: 10, bottom: 5),
-        child: Text(bottomText, style: const TextStyle(fontSize: 15)),
-      ),
-    ],
-  );
+        child: Column(
+            crossAxisAlignment:
+                CrossAxisAlignment.start, // Set cross axis alignment to start (left-aligned)
+
+            children: [
+              const Text(
+                "Error details:",
+                style: TextStyle(fontSize: 15),
+              ),
 
-  // Add "Do not close the app" while order is pending
-  if (pendingOrder.state == PendingOrderState.submitting ||
-      pendingOrder.state == PendingOrderState.submittedSuccessfully) {
-    body.children.add(
-      const Padding(
-        padding: EdgeInsets.only(left: 10, right: 10, bottom: 5),
-        child: Text("Do not close the app!",
-            style: TextStyle(fontSize: 15, fontWeight: FontWeight.bold)),
+              Padding(
+                padding: const EdgeInsets.fromLTRB(0, 15.0, 0, 15.0),
+                child: Container(
+
+                  color: Colors.grey.shade300,
+                  child: Text(
+                    getPrettyJSONString(pendingOrder.failureReason?.details ?? ""),
```

Comment:
nit: it would be nice if the error text could be copied. Resp. If we'd add a share logs button.